providers/azure/metrics.go

```diff
@@ -0,0 +1,186 @@
+package azure
+
+import (
+	"strings"
+	"time"
+
+	"github.com/pkg/errors"
+	"github.com/virtual-kubelet/virtual-kubelet/providers/azure/client/aci"
+	"golang.org/x/sync/errgroup"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	stats "k8s.io/kubernetes/pkg/kubelet/apis/stats/v1alpha1"
+)
+
+const startTimeFormat = "2006-01-01 15:04:05 -0700 MST"
```

Comment:
-01-02

Reply:
Ah nice catch, that would be why I'm getting negative uptime :)